build.gradle

```diff
@@ -38,11 +36,10 @@ subprojects {
 	tasks.withType(Test) {
 		testlogger {
 			theme = 'mocha'
-			showStandardStreams = false
-			showSkippedStandardStreams = false
+			showStandardStreams = true
```

Comment:
✏️ I might turn these back to false. 
It turns out the logging isn't as great as I hoped.